crates/sage/src/scoring.rs

```diff
@@ -440,6 +462,9 @@ impl<'db> Scorer<'db> {
                 poisson = 1E-325;
             }
 
+            // Sage operates on masses without protons; [M] instead of [MH+]
+            let mz = precursor[precursor_index].mz - PROTON;
```

Comment:
[self q] would it be worth to cache the mzs in a heap allocation? ...

Reply:
answer ... nope, used only once if report psms==1

Reply:
Definitely not, heap allocations are generally very expensive!